radixdlt-core/radixdlt/src/main/java/com/radixdlt/RadixNodeModule.java

```diff
@@ -296,5 +297,11 @@ protected void configure() {
     var enableTransactions = properties.get("api.transactions.enable", false);
     var enableSign = properties.get("api.sign.enable", false);
     install(new ApiModule(bindAddress, port, enableTransactions, enableSign));
+
+    // SubState Hash Accumulator
```

Comment:
perhaps we should stick to `Substate` (a single word)?

Reply:
IntelliJ complains about that, but seems we use it like you said throughout the code, right?

Reply:
yes

Reply:
done